src/main_thread/api/debug/render.ts

```diff
@@ -35,6 +39,9 @@ export default function renderDebugElement(
   debugWrapperElt.style.bottom = "0px";
   parentElt.appendChild(debugWrapperElt);
   cancelSignal.register(() => {
+    isUsed.setValue(false);
     parentElt.removeChild(debugWrapperElt);
   });
 }
+
+renderDebugElement.__IS_USED = isUsed;
```

Comment:
I'm not sure about this solution but I didn't want to change the API for `createDebugElement` that return only a function for now.

In terms of logic it's working only if there is maximum 1 debug element at a time.

Reply:
I have removed it, the debug Element now subscribe to changes with `_priv_subscribeToSegmentSinkMetrics`